lib/handbook/addon/docs/components/ancestry-display/demo-ancestry-display/template.hbs

```diff
@@ -0,0 +1,49 @@
+{{#docs-demo as |demo|}}
+    {{#demo.example name='ancestry-display.ancestry-display.hbs'}}
+        {{ancestry-display node=currentNode useLinks=useLinks delimiter=currentDelimiter}}
+    {{/demo.example}}
+
+
+    {{#bs-button class='center-block m-b-lg' onClick=(action (mut useLinks) (not useLinks))}}
+        set: useLinks = {{not useLinks}}
+    {{/bs-button}}
+    {{#bs-dropdown as |dd|}}
+        {{#dd.button}}
+            Delimiter
+            {{fa-icon 'caret-down'}}
+        {{/dd.button}}
+        {{#dd.menu local-class='demo-list'}}
+            {{#each delimiters as |delimiter|}}
+                <li role="menuitem">
+                    <a local-class='demo-link'
+                        role="button"
+                        {{action 'setDelimiter' delimiter}}
+                    >
+                        {{ delimiter }}
+                    </a>
+                </li>
+            {{/each}}
+        {{/dd.menu}}
+    {{/bs-dropdown}}
+    {{#bs-dropdown as |dd|}}
+        {{#dd.button}}
+            # of ancestors
+            {{fa-icon 'caret-down'}}
+        {{/dd.button}}
+        {{#dd.menu local-class='demo-list'}}
+            {{#each nodes as |node index|}}
+                <li role="menuitem">
+                    <a local-class='demo-link'
+                        role="button"
+                        {{action 'setCurrentNode' index}}
+                    >
+                        {{ index }}
+                    </a>
+                </li>
+            {{/each}}
+        {{/dd.menu}}
+    {{/bs-dropdown}}
+
+
+    {{demo.snippet 'ancestry-display.ancestry-display.hbs'}}
+{{/docs-demo}}
```

Comment:
While this interactive demo is cool, and I appreciate the effort, I think it would actually be better to show specific examples instead. A good rule of thumb is that if a component parameter is something that would normally change due to user interaction in real-world usage (e.g. hiding a notification) then adding an interactive control in the handbook makes sense, otherwise, if the parameters would normally be static or based on data, specific examples demonstrating various usages would be better.